.github/workflows/deploy.yml

```
@@ -1,15 +1,17 @@
 name: deploy
 
 on:
-  push:
+  workflow_run:
+    workflows: ["spellcheck", "lintcheck"]
+    types: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)